src/constants.ts

```diff
@@ -155,6 +155,11 @@ export const AUTHORITY_TOKEN_DATA = TOKEN_AUTHORITY_MASK | 1;
  */
 export const HATHOR_TOKEN_CONFIG = { name: 'Hathor', symbol: 'HTR', uid: '00' };
```

Comment:
```suggestion
export const HATHOR_TOKEN_CONFIG = { name: 'Hathor', symbol: 'HTR', uid: HATHOR_TOKEN_UID };
```

Reply:
I don't this this is an improvement since this config will not be on the constants for long